internal/jwt_utils.go

```diff
@@ -13,36 +13,96 @@ import (
 	"github.com/golang-jwt/jwt"
 )
 
-// TODO: get this in a more secure manner not hard-coded in the application.
+// TODO: get these in a more secure manner not hard-coded in the application.
 // See https://github.com/Lasagna-Love-Portal/bechamel-api/issues/5
-var jwtSigningKey = []byte("GetThisFromENV")
+var jwtAccessSigningKey = []byte("GetThisFromENV")
```

Comment:
I'm not a huge fan of global variables. Not a blocker here, just want to flag it for later.